ember_debug/view-debug.js

```diff
@@ -2,7 +2,7 @@
 import PortMixin from "ember-debug/mixins/port-mixin";
 
 const Ember = window.Ember;
-const { guidFor, $, computed, run, Object: EmberObject, typeOf, Component, Controller, ViewUtils, A, LinkComponent, String } = Ember;
+const { guidFor, $, computed, run, Object: EmberObject, typeOf, Component, Controller, ViewUtils, A, LinkComponent, String, View } = Ember;
```

Comment:
just to clarify: should I remove `|| View.views` instead? the issue was that `View` is undefined since it's not imported from anywhere or defined as a global.